docs/getting-started.md

```diff
@@ -95,26 +87,39 @@ accessibility of open software.
   <source src="/img/macOS-open.mp4" type="video/mp4">
 </video>
 
+If you are installing on a [Mac computer with Apple Silicon ][as] please note
+that you may experience issues as Radicle Upstream has not been tested on the
+Apple Silicon platform.
+
+<blockquote>
+While we will be participating in the Apple Developer Program to ensure
+accessibility to our macOS users, we do not agree with Apple's financial
+gatekeeping and increasing control over application distribution on their
+platform. Forcing developers to "pay-to-play" under the guise of a security
+solution is an extractive form of vendor lock-in that inhibits the growth and
+accessibility of open software.
+</blockquote>
+
+
 ## Creating an identity
 
-To create an identity on the Radicle network, you can use the Upstream app. The
-app will generate a unique identifier — known as a [**Device ID**][di] — that's
-paired to the computer you've installed Radicle on. For now, this Device ID is
+To create an identity on the Radicle network, you can use the Upstream client. After onboarding, it will generate a unique identifier — known as a [**Device ID**][di] — that's
```

Comment:
I find this a little confusing. It sounds as if creating the Radicle Identity via Upstream is an alternative way of doing it.

I'd expect this `Creating an identity` section to be `Getting started`, where we go on informing the user about the onboarding and what happens in that stage.

Reply:
I see what you mean, I think the goal is for the Upstream client to be one of the alternatives for creating an identity but not the only one. I'll rephrase the language for now.

Reply:
```suggestion
To create an identity on the Radicle network, use the Radicle Upstream app. After choosing a display name and setting a passphrase, it will generate a unique identifier — known as a [**Device ID**][di] — that's
```